src/yadism/esf/esf.py

```diff
@@ -116,11 +116,12 @@ def compute_local(self):
         if self._computed:
             return
         cfc = cf.Combiner(self)
-        full_orders = [(o, 0, 0, 0) for o in self.orders]
         # prepare scale variations
         sv_manager = self.info.configs.managers["sv_manager"]
         if sv_manager is not None:
             full_orders = sv.build_orders(self.info.configs.theory["pto"])
```

Comment:
as @andreab1997 pointed out: this should be `"pto_evol"`. Since FONLL-B is a NLO scheme we also need to have NLO scale variations. Actually, this means that also the `massive` grids need to have that setting!